src/index.html

```diff
@@ -7,8 +7,48 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="container">
+      <a href="#home" class="logo">
+        <img src="images/logo.png" alt="Logo">
+      </a>
+      <nav>
+        <ul class="nav_list">
+          <li class="nav_item">
+            <a href="#" class="box is-active">Apple</a>
+          </li>
+          <li class="nav_item">
+            <a href="#" class="box">Samsung</a>
+          </li>
+          <li class="nav_item">
+            <a href="#" class="box">Smartphones</a>
+          </li>
+          <li class="nav_item">
+            <a href="#" 
+            class="box" 
+            data-qa="hover"
+            >
+            Laptops & Computers
+            </a>
```

Comment:
```suggestion
            <a 
              href="#" 
              class="box" 
              data-qa="hover"
            >
              Laptops & Computers
            </a>
```

One more problem: 
If the HTML-element has long attribute values or number of attributes is more than 2 - start each one, **including the first**, on the new line with 2-space indentation related to tag. (Checklist p13)